src/ape/api/accounts.py

```diff
@@ -258,7 +263,7 @@ def declare(self, contract: "ContractContainer", *args, **kwargs) -> ReceiptAPI:
 
     def check_signature(
         self,
-        data: Union[SignableMessage, TransactionAPI],
+        data: Union[SignableMessage, TransactionAPI, str, EIP712Message],
```

Comment:
should be bytes (hex bytes) also be handled?

Reply:
I just don't think this should be a part of the diff for this refactor